plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -45,6 +50,26 @@
     private String requiredReplicaSetName;
     private String implicitRowFieldPrefix = "_pos";
 
+    @AssertTrue(message = "Exactly one of 'mongodb.schema-database' or 'mongodb.schema-collection' must be specified, or the default _schema could be created in the same database")
+    public boolean isSchemaManagementConfigValid()
+    {
+        return schemaDatabase.isEmpty() || schemaCollection.equals("_schema");
```

Comment:
I'm losing the context. Why do we need this limitation?

Reply:
This check we added because we can either use the _schema default trino implementation or external database implementation. We can remove the check and say that preference is given to external database if specified or throw error while parsing the properties.